src/modules/microdds_client/dds_topics.yaml

```diff
@@ -0,0 +1,96 @@
+#####
+#
+# This file maps all the topics that are to be used on the micro DDS client.
+#
+#####
+publications:
+  - topic: /fmu/out/vehicle_status
+    type: px4_msgs::msg::VehicleStatus
+
+  - topic: /fmu/out/failsafe_flags
+    type: px4_msgs::msg::FailsafeFlags
+
+  - topic: /fmu/out/trajectory_waypoint
+    type: px4_msgs::msg::TrajectoryWaypoint
+
+  - topic: /fmu/out/vehicle_control_mode
+    type: px4_msgs::msg::VehicleControlMode
+
+  - topic: /fmu/out/vehicle_odometry
+    type: px4_msgs::msg::VehicleOdometry
+
+  - topic: /fmu/out/vehicle_trajectory_waypoint_desired
+    type: px4_msgs::msg::VehicleTrajectoryWaypoint
+
+  - topic: /fmu/out/collision_constraints
+    type: px4_msgs::msg::CollisionConstraints
+
+  - topic: /fmu/out/timesync_status
+    type: px4_msgs::msg::TimesyncStatus
+
+  - topic: /fmu/out/sensor_combined
+    type: px4_msgs::msg::SensorCombined
+
+  - topic: /fmu/out/vehicle_attitude
+    type: px4_msgs::msg::VehicleAttitude
+
+  - topic: /fmu/out/vehicle_local_position
+    type: px4_msgs::msg::VehicleLocalPosition
+
+  - topic: /fmu/out/vehicle_global_position
+    type: px4_msgs::msg::VehicleGlobalPosition
+
+  - topic: /fmu/out/vehicle_gps_position
+    type: px4_msgs::msg::SensorGps
+
+  # - topic: /fmu/out/vehicle_angular_velocity
+  #   type: px4_msgs::msg::VehicleAngularVelocity
+
+subscriptions:
+  - topic: /fmu/in/debug_array
+    type: px4_msgs::msg::DebugArray
+
+  - topic: /fmu/in/debug_key_value
+    type: px4_msgs::msg::DebugKeyValue
+
+  - topic: /fmu/in/debug_value
+    type: px4_msgs::msg::DebugValue
+
+  - topic: /fmu/in/debug_vect
+    type: px4_msgs::msg::DebugVect
+
+  - topic: /fmu/in/offboard_control_mode
+    type: px4_msgs::msg::OffboardControlMode
+
+  - topic: /fmu/in/sensor_optical_flow
+    type: px4_msgs::msg::SensorOpticalFlow
+
+  - topic: /fmu/in/position_setpoint_triplet
+    type: px4_msgs::msg::PositionSetpointTriplet
+
+  - topic: /fmu/in/telemetry_status
+    type: px4_msgs::msg::TelemetryStatus
+
+  - topic: /fmu/in/vehicle_command
+    type: px4_msgs::msg::VehicleCommand
+
+  - topic: /fmu/in/trajectory_setpoint
+    type: px4_msgs::msg::TrajectorySetpoint
+
+  - topic: /fmu/in/vehicle_mocap_odometry
+    type: px4_msgs::msg::VehicleOdometry
+
+  - topic: /fmu/in/vehicle_visual_odometry
+    type: px4_msgs::msg::VehicleOdometry
+
+  - topic: /fmu/in/vehicle_trajectory_waypoint
+    type: px4_msgs::msg::VehicleTrajectoryWaypoint
+
+  - topic: /fmu/in/onboard_computer_status
+    type: px4_msgs::msg::OnboardComputerStatus
+
+  - topic: /fmu/in/trajectory_bezier
+    type: px4_msgs::msg::TrajectoryBezier
+
+  - topic: /fmu/in/vehicle_trajectory_bezier
+    type: px4_msgs::msg::VehicleTrajectoryBezier
```

Comment:
Could we add `px4_msgs::msg::VehicleRatesSetpoint` and `px4_msgs::msg::VehicleAttitudeSetpoint` to the subscription topic list?

Reply:
Done.